ffi/capi/bindings/js/CollatorOptions.d.ts

```diff
@@ -0,0 +1,39 @@
+// generated by diplomat-tool
+import type { CollatorAlternateHandling } from "./CollatorAlternateHandling"
+import type { CollatorBackwardSecondLevel } from "./CollatorBackwardSecondLevel"
+import type { CollatorCaseFirst } from "./CollatorCaseFirst"
+import type { CollatorCaseLevel } from "./CollatorCaseLevel"
+import type { CollatorMaxVariable } from "./CollatorMaxVariable"
+import type { CollatorNumeric } from "./CollatorNumeric"
+import type { CollatorStrength } from "./CollatorStrength"
+import type { pointer, char } from "./diplomat-runtime.d.ts";
+
+
+/** See the [Rust documentation for `CollatorOptions`](https://docs.rs/icu/latest/icu/collator/struct.CollatorOptions.html) for more information.
+*/
+export class CollatorOptions {
+    #strength;
```

Comment:
nit: Private variables should not be present in .d.ts files for structs, I need to go back and fix this in the templates file.